backend/apps/residents/migrations/0002_studentprofile.py

```
@@ -0,0 +1,46 @@
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('residences', '0002_delete_studentprofile'),
+        ('residents', '0001_initial'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)